ERROR - 2022-10-03 10:20:06 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', 'root', 'ci4_morut', 3306, '', 0)
#1 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\morut\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\morut\app\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(907): App\Controllers\Welcome->index()
#8 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
#9 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\morut\public\index.php(55): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\morut\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\morut\app\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(907): App\Controllers\Welcome->index()
#7 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
#8 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\morut\public\index.php(55): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-10-03 10:20:06 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `gis_batas_kecamatan`', [], false)
 3 SYSTEMPATH\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Welcome->index()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-03 10:21:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', 'root', 'ci4_morut', 3306, '', 0)
#1 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\morut\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\morut\app\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(907): App\Controllers\Welcome->index()
#8 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
#9 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\morut\public\index.php(55): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\morut\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\morut\app\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(907): App\Controllers\Welcome->index()
#7 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
#8 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\morut\public\index.php(55): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-10-03 10:21:11 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `gis_batas_kecamatan`', [], false)
 3 SYSTEMPATH\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Welcome->index()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-03 10:21:14 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', 'root', 'ci4_morut', 3306, '', 0)
#1 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\morut\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\morut\app\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(907): App\Controllers\Welcome->index()
#8 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
#9 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\morut\public\index.php(55): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\morut\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\morut\app\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(907): App\Controllers\Welcome->index()
#7 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
#8 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\morut\public\index.php(55): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-10-03 10:21:14 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `gis_batas_kecamatan`', [], false)
 3 SYSTEMPATH\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Welcome->index()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
ERROR - 2022-10-03 10:21:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', 'root', 'ci4_morut', 3306, '', 0)
#1 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\morut\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\morut\app\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(907): App\Controllers\Welcome->index()
#8 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
#9 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\morut\public\index.php(55): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\laragon\www\morut\vendor\codeigniter4\framework\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#4 C:\laragon\www\morut\vendor\codeigniter4\framework\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\laragon\www\morut\app\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
#6 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(907): App\Controllers\Welcome->index()
#7 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
#8 C:\laragon\www\morut\vendor\codeigniter4\framework\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\laragon\www\morut\public\index.php(55): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-10-03 10:21:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `gis_batas_kecamatan`', [], false)
 3 SYSTEMPATH\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Welcome.php(26): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Welcome->index()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Welcome))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-10-03 10:23:14 --> unlink(img/gis/facilities/1664809292_4830f044c934ae9a51c0.png): No such file or directory
in APPPATH\Controllers\Gis_facilities.php on line 302.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(img/gis/facilities/1664809292_4830f044c934ae9a51c0.png): No such file or directory', 'APPPATH\\Controllers\\Gis_facilities.php', 302, [...])
 2 APPPATH\Controllers\Gis_facilities.php(302): unlink('img/gis/facilities/1664809292_4830f044c934ae9a51c0.png')
 3 SYSTEMPATH\CodeIgniter.php(907): App\Controllers\Gis_facilities->delete('12')
 4 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Gis_facilities))
 5 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
